frontend/test/metabase/scenarios/onboarding/setup/setup.cy.spec.js

```diff
@@ -121,14 +121,14 @@ describe("scenarios > setup", () => {
       cy.findByText("Show more options").click();
       cy.findByText("H2").click();
       cy.findByLabelText("Display name").type("Metabase H2");
-      cy.findByText("Next")
+      cy.findByText("Connect database")
```

Comment:
Modify this to reflect the UI change.
![image](https://user-images.githubusercontent.com/1937582/169349022-c43f9fed-3f73-4f84-b93a-b673f92a36f7.png)